package.json

```diff
@@ -3,41 +3,33 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^4.11.3",
+    "@material-ui/core": "4.12.3",
```

Comment:
Are we intentionally locking to a specific version?

Ok if we are, just want to be intentional

Reply:
Even this update broke the front end, only small naming changes were needed but still, updates might require time/work. For this reason I would lock the version in to encourage the next dev to read the changelog before updating this package.